#!/usr/bin/env python
PACKAGE = "dynamic_reconfigure_test"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("H_min", int_t, 0, "h min parameter", 0, 0, 180)
gen.add("H_max", int_t, 0, "h max parameter", 180, 0, 180)
gen.add("S_min", int_t, 0, "s min parameter", 0, 0, 255)
gen.add("S_max", int_t, 0, "s max parameter", 255, 0, 255)
gen.add("V_min", int_t, 0, "v min parameter", 0, 0, 255)
gen.add("V_max", int_t, 0, "v max parameter", 255, 0, 255)

gen.add("SendData", str_t,0, "A SendData parameter" , "")
gen.add("ReceiveData", str_t,0, "A ReceiceData parameter", "")

exit(gen.generate(PACKAGE, "dynamic_reconfigure_test", "dyn_reconfig_test"))
